CMakeLists.txt

```diff
@@ -12,8 +12,8 @@ include(CTest)
 
 rocm_setup_version(VERSION 0.5.0)
 
-find_path(OPENCL_ROOT lib/x86_64/bitcode/opencl.amdgcn.bc
-    PATH_SUFFIXES 
+find_path(OPENCL_ROOT lib/x86_64/libOpenCL.so.1
```

Comment:
Why is this even needed? It never used.

Reply:
You will use the OpenCL's bin/clinfo, should I look for that instead?

Reply:
I see, that is just to infer the target name. We could just remove that and use the `llvm-dis` path instead.

Reply:
Okay I will remove the usage of OpenCL

Reply:
Sorry, it seems you will use the OPENCL_ROOT inside of clang-ocl.in

Reply:
Yea thats right, we need the `opencl-c.h` header.